### PR TITLE
seller item screen created

### PR DESCRIPTION
**Pull request by adisa39**
Seller Item UI created with Trust Ometer component for your review

### CODE REVIEW

File: src/app/[locale]/seller/product/page.tsx
Comment (swoocn) [inline src/app/[locale]/seller/product/page.tsx:1]: The product screen looks nice but isn't a PiFest scope requirement.

image

File: src/app/[locale]/seller/seller-item/page.tsx
Comment (swoocn) [inline src/app/[locale]/seller/seller-item/page.tsx:1]: Great job on the screen!  Please refer to updated screen requirements @ https://sharing.clickup.com/9015444408/t/h/86byq700x/A7DI9S5MNNJW8YA per UX team.  Thx!

**Comment by swoocn** [comment]
@adisa39 - The **Buy From Seller** screen looks great!  Please refer to updated screen requirements @ https://sharing.clickup.com/9015444408/t/h/86byq700x/A7DI9S5MNNJW8YA.  The expected screen will now look like ->

image

**Review by swoocn — CHANGES_REQUESTED** [review]
New screen requirements @ https://sharing.clickup.com/9015444408/t/h/86byq700x/A7DI9S5MNNJW8YA.

File: src/app/[locale]/seller/seller-item/page.tsx
Comment (adisa39) [reply src/app/[locale]/seller/seller-item/page.tsx:1]: Alright👍

**Review by hollusam — APPROVED** [review]

**Comment by swoocn** [comment]
GJ with the adjustments, @adisa39!  
Please see the following notes based off the latest design comments (reference @  https://www.figma.com/design/gSxgy18iMqrs9JorIaznCV/pi_commerce_app_mockup?node-id=1054-3828&t=F8HKgQfnc0YOBEHy-1) ->

image

Thanks again!

File: src/components/shared/Review/emojipicker.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+interface EmojiPickerProps {
+    clickDisabled?: boolean;
+    reviews?: { [key: string]: number };
+}
+
+export default function EmojiPicker(props: EmojiPickerProps) {
+
+    const despairEmoji: Emoji = { name: "Despair", unicode: "🙂", code: ":despair:", value: 1 };
```
Comment (swoocn) [inline src/components/shared/Review/emojipicker.tsx]: Can you please weigh the value of Despair to 0?  Thanks!

**Comment by swoocn** [comment]
@adisa39 - We're almost there!
The UX team has finalized the design per below.  Could you please make the adjustments when you can?  Thanks for being so flexible!

image

A few details to note in the updated screenshot ->
- Only the first letter in each sub-header should be in uppercase.
- Please evenly align and center the emoji rating feature. 
- Please also remove the Map of Pi logo from the Seller Address section.

**Review by swoocn — APPROVED** [review]

**Pull request merged**